components/HeroBanner.tsx

```diff
@@ -0,0 +1,42 @@
+// CSS
+import styles from '../styles/components/HeroBanner.module.scss';
+import Button, { ButtonType } from './Button';
+import { HeroIcons } from '../config.yaml';
+import Image from '../components/Image';
+
+interface Props {
+  imagePath: string;
+  altText: string;
+}
+
+export default function HeroBanner({ imagePath, altText }: Props) {
+  return (
+    <div className={styles.HeroBanner}>
+      {/* Left Column */}
+      <div className={styles.leftColumn}>
+        <Image src={imagePath} alt={altText} width={100} height={100} />
+        <Button type={ButtonType.LIGHT}>Apply</Button>
+      </div>
+      {/* Right Column */}
+      <div className={styles.rightColumn}>
+        <h1>Trent Computer Science Club Association</h1>
+        <hr />
+        <div className={styles.tagline}>Where passion meets code</div>
+        <div className={styles.socialMedia}>
+          {HeroIcons.map((icon) => (
```

Comment:
```suggestion
          {heroIcons.map((icon) => (
```

Alternatively we can switch `types.d.ts` to be `HeroIcons`